### PR TITLE
Adding support for ScyllaDB clusters in AWS

### PR DESCRIPTION
**Pull request by blarghmatey**
Added a cloud profile and orchestrate state, along with configuration deployment and tests for ScyllaDB

### CODE REVIEW

File: salt/orchestrate/services/scylladb.sls [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+format_data_drive:
+  salt.function:
+    - tgt: 'G@roles:scylladb and G@environment:{{ ENVIRONMENT }}'
+    - tgt_type: compound
+    - name: state.single
+    - arg:
+        - blockdev.formatted
+    - kwarg:
+        name: /dev/xvdb
```
Comment (shaidar) [inline salt/orchestrate/services/scylladb.sls]: Isn't the drive /dev/xvda2?

File: salt/orchestrate/services/scylladb.sls [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+mount_data_drive:
+  salt.function:
+    - tgt: 'G@roles:scylladb and G@environment:{{ ENVIRONMENT }}'
+    - tgt_type: compound
+    - name: state.single
+    - arg:
+        - mount.mounted
+    - kwarg:
+        name: /var/lib/scylla
+        device: /dev/xvdb
```
Comment (shaidar) [inline salt/orchestrate/services/scylladb.sls]: Isn't the drive /dev/xvda2?

File: salt/orchestrate/services/scylladb.sls (hunk 1 above)
Comment (blarghmatey) [reply salt/orchestrate/services/scylladb.sls]: No, xvda2 would be partition 2 on the xvda block device.

File: salt/orchestrate/services/scylladb.sls (hunk 2 above)
Comment (blarghmatey) [reply salt/orchestrate/services/scylladb.sls]: No, xvda2 would be partition 2 on the xvda block device.

**Comment by shaidar** [comment]
:thumbsup:

**Pull request merged**